source/includes/gho/_api.md

```diff
@@ -74,332 +447,1037 @@ last | [Int](#int) | Returns the last _n_ elements from the list. |
+        }
+      }
+    }
+  }
+}
+
+```
+
+```graphql
+# Request employees that have a date value between "2017-04-13" and "2018-04-13" (exclusive) for the
+# "1_year_anniversary" Custom Field. For each of these employees, return their ID and
+# the value for the "1_year_anniversary" Custom Field (and only the value for this Custom Field).
+{
+  employees(
+    first: 25,
+    customFieldValues: [{ id: "1_year_anniversary", dateValue: { after: "2017-04-13", before: "2018-04-13" } }]
+  ) {
+    pageInfo {
+      hasNextPage
+      endCursor
+    }
+    edges {
+      node {
+        id
+        customFieldValues(customFieldIds: ["1_year_anniversary"]) {
+          customField {
+            id
+          }
+          value
+        }
+      }
+    }
+  }
+}
+
+```
+
+```graphql
+# Request employees that have Employee 35 or Employee 40 set as the value for the "mentor" Custom Field. For each of
+# these employees, return their id, work email address, and "about me" text.
+{
+  employees(
+    first: 25,
+    customFieldValues: [{id: "mentor", idValues: [35, 40]}]
+  ) {
+    pageInfo {
+      hasNextPage
+      endCursor
+    }
+    edges {
+      node {
+        id
+        email
+        about
+      }
+    }
+  }
+}
+
+```
+
+```graphql
+# Request only those employees that have a dateOfBirth between 1989-12-31 and 2000-01-01.
+# These dates are exclusive (e.g. someone who was born on 1989-12-31 or 2000-01-01 would not be included.
+{
+  employees(first: 25, dateOfBirthFilter: { dateFilter: { after: "1989-12-31", before: "2000-01-01" } }) {
+    pageInfo {
+      hasNextPage
+      endCursor
+    }
+    edges {
+      node {
+        id
+        email
+        startDate
+      }
+    }
+  }
+}
+
+```
+
+```graphql
+# Request only those employees that have a department set (department is not null). For each employee that fits the criteria,
+# return their id and work email
+{
+  employees(first: 25, departmentFilter: { anyValue: true }) {
+    pageInfo {
+      hasNextPage
+      endCursor
+    }
+    edges {
+      node {
+        id
+        email
+      }
+    }
+  }
+}
+
+```
+
+```graphql
+# Request only those employees that lack a department (department is null). For each employee that fits the criteria,
+# return their id and work email
+{
+  employees(first: 25, departmentFilter: { noValue: true }) {
+    pageInfo {
+      hasNextPage
+      endCursor
+    }
+    edges {
+      node {
+        id
+        email
+      }
+    }
+  }
+}
+
+```
+
+```graphql
+# Request only those employees that are in a specific department(s). For each employee that fits the criteria,
+# return their id and work email
+{
+  employees(first: 25, departmentFilter: { departmentIds: [1, 3] }) {
+    pageInfo {
+      hasNextPage
+      endCursor
+    }
+    edges {
+      node {
+        id
+        email
+      }
+    }
+  }
+}
+
+```
+
+```graphql
+# Request only those employees that have specific emails(s). For each employee that fits the criteria,
```

Comment:
I don't think we need the (s) after emails since its already plural. It would also look weird doing `have a specific email(s)` or `have specific email(s)`

Reply:
fixed!